### PR TITLE
Merge master nuget commits -> xplat

### PR DESCRIPTION
**Pull request by AndyGerlicher**
I did this real quick so we can get it on the radar. @tmeschter, thoughts? I basically did take target on the nuspec. Do we want to do something different? Or do you want to do the merge and get these full framework packages producing in xplat now (might be easier now than later)? 

Pretty much every file from each commit had a conflict though so it would be good to get on this one.

### CODE REVIEW

**Comment by tmeschter** [comment]
@AndyGerlicher Does a build of xplat already produce both Desktop and .Net Standard binaries?

**Comment by rainersigwald** [comment]
@tmeschter Yes, xplat builds both since #850.

**Comment by AndyGerlicher** [comment]
@dotnet-bot test this please

File: src/dirs.proj [hunk 1]
```diff
@@ -58,7 +58,11 @@
     </TraversalBuildDependsOn>
   </PropertyGroup>
 
-
-
+  <!-- Workaround to remove items from PackagesNuSpecFiles (defined in packages.targets) -->
```
Comment (AndyGerlicher) [inline src/dirs.proj]: Added this to stop trying to create packages those projects on the NetCore build.

Comment (rainersigwald) [inline src/dirs.proj]: Maybe also comment that they're added via wildcard there, so there's not a great way to remove them otherwise?

**Comment by rainersigwald** [comment]
:shipit:

**Comment by brthor** [comment]
When the nuget package for this build becomes available, will the item remove syntax be supported outside targets in xplat msbuild?

**Pull request merged**